packages/store/addon/-private/system/fetch-manager.ts

```diff
@@ -339,14 +349,17 @@ export default class FetchManager {
 
   rejectFetchedItems(seeking: { [id: string]: PendingFetchItem }, snapshots: Snapshot[], error?) {
     for (let i = 0, l = snapshots.length; i < l; i++) {
-      let identifier = snapshots[i];
-      let pair = seeking[identifier.id];
+      let snapshot = snapshots[i];
+      // TODO refactor to identifier.lid to avoid this cast to string
+      //  we can do this case because you can only fetch an identifier
+      //  that has an ID
```

Comment:
Same as above, we can because that matching is done elsewhere, here we are dealing with Snapshots and identifiers. Id usage is incidental and not required at this layer.

Reply:
SAme here